### ISSUE TITLE
[ui] Use `right-click`+`scroll` as a way to zoom in/out of the DAG in the UI

### ISSUE THREAD

**Issue by mkleinbort-ic** [dagster-io/dagster#18027]
### What's the use case?

Sometimes it's nice to navigate the DAG just with the mouse - this would be easier if one could use `right-click`+`scroll` as a way to zoom in/out of the DAG.

### Message from the maintainers

Impacted by this issue? Give it a 👍! We factor engagement into prioritization.

**Comment by salazarm** [comment]
You can hold shift while scrolling to zoom in and out

**Comment by mkleinbort-ic** [comment]
I know, it's what I do currently - but my left hand is often busy with a cup of coffee (only half joking)

**Comment by salazarm** [comment]
If you're trying to get to a parent node or sibling node you might find it easier to use our keyboard navigation with the arrow keys? (Next release also adds keyboard navigation for the sidebar so that will work there too).

**Comment by mkleinbort-ic** [comment]
No, I'm trying to replicate the "pinch to zoom" trackpad functionality using a mouse. 
Holding right click + scroll bar is a standard way to zoom in/out in some software:

Autodesk Maya (3D Modeling)
Blender (3D Modeling)
SketchUp (3D Modeling)
Adobe Photoshop (Image Editing)
Adobe Illustrator (Graphic Design)
CorelDRAW (Graphic Design)
GIMP (Image Editing)
AutoCAD (CAD Software)
SolidWorks (CAD Software)
